crates/oxc_parser/src/js/binding.rs

```diff
@@ -90,8 +92,11 @@ impl<'a> Parser<'a> {
             //       ^ BindingIdentifier
             if let PropertyKey::Identifier(ident) = &key {
                 shorthand = true;
-                let binding_identifier =
-                    BindingIdentifier { span: ident.span, name: ident.name.clone() };
+                let binding_identifier = BindingIdentifier {
+                    span: ident.span,
+                    name: ident.name.clone(),
+                    symbol_id: Cell::default(),
+                };
```

Comment:
nitpick: we can add `fn new` to BindingIdentifier and IdentifierReference, since this is getting repeated all over the place.